docs/src/main/sphinx/release/release-478.md

```diff
@@ -0,0 +1,80 @@
+# Release 478 (22 Oct 2025)
+
+## General
+
+* Add support for column lineage in `UNNEST` clauses. ({issue}`16946`)
+* Add `retry-policy.allowed` configuration property to specify which query retry
```

Comment:
```
* Add support for limiting which retry policies a user can select. This can be configured using the `retry-policy.allowed` option. ({issue}`26628`)
```